src/bebone/core/types/matrix/matrix_2x2.cpp

```diff
@@ -0,0 +1,36 @@
+#include "matrix_2x2.h"
+
+namespace bebone::core {
+    Matrix<f32, 2, 2>::Matrix(f32 n00, f32 n01, f32 n10, f32 n11) {
```

Comment:
Just question, arguments are not const refs just cause there is a lot of them ?

Reply:
yeah I somehow forgot about const refs... I will add them